timed/projects/filters.py

```diff
@@ -24,7 +24,40 @@ class ProjectFilterSet(FilterSet):
     """Filter set for the projects endpoint."""
 
     archived = NumberFilter(field_name="archived")
-    reviewer = NumberFilter(field_name="reviewers")
+    has_manager = NumberFilter(method="filter_has_manager")
+    has_reviewer = NumberFilter(method="filter_has_reviewer")
+
+    def filter_has_manager(self, queryset, name, value):
+        if not value:  # pragma: no cover
+            return queryset
+        return queryset.filter(
+            Q(
+                pk__in=models.ProjectAssignee.objects.filter(
+                    is_manager=True, user_id=value
+                ).values("project_id"),
+            )
+            | Q(
+                customer_id__in=models.CustomerAssignee.objects.filter(
+                    is_manager=True, user_id=value
+                ).values("customer_id"),
+            )
+        )
+
+    def filter_has_reviewer(self, queryset, name, value):
```

Comment:
Isn't the method `filter_has_reviewer` exactly the same as the method `filter_has_manager`?

Reply:
No, `filter_has_manager` filters assignees with the manager role and `filter_has_reviewer` filters assignees with the reviewer role. One has `is_manager=True` and the other `is_reviewer=True`.

Reply:
Ah ok I can see it now :wink: